src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -1811,22 +1811,12 @@ private void DecorateItemsWithMetadataFromProjectItemElement(ProjectItemElement
                     include.Add(itemElement.Include);
```

Comment:
These 3 lines can be removed, since `itemsAndMetadataFromInclude` is not being used anymore.